kalite/topic_tools/__init__.py

```diff
@@ -430,7 +430,7 @@ def get_exercise_data(request, exercise_id=None):
     if not exercise:
         return None
 
-    exercise_root = os.path.join(settings.KHAN_EXERCISES_DIRPATH, "exercises")
+    exercise_root = os.path.join(settings.KHAN_EXERCISES_DIRPATH)
```

Comment:
I don't believe this edit is necessary. It should be looking inside the exercises folder.